packages/app/components/Queue/TA/TAQueueList.tsx

```diff
@@ -259,16 +262,46 @@ const NoQuestionsText = styled.div`
   color: #212934;
 `;
 
+function NotifReminderButton() {
+  const [isNotifOpen, setIsNotifOpen] = useState(false);
+  const NotifRemindText = styled(Button)`
```

Comment:
I like that you included some styling with this button! For consistency, it would be nice if the button was styled the same as the "Edit Queue Details" button on the side: https://github.com/sandboxnu/office-hours/blob/283996ea793cba7fc69f83e645fd7a0a8e511839/packages/app/components/Queue/TA/TAQueueList.tsx#L77